BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
+systems. Other operating systems have reported to work, but support is not
+actively maintained by the FreeOrion developers. FreeOrion is developed for x86
+compatible processor architectures, other architectures haven't been tested.
+
+The FreeOrion source code makes heavy use of templates and requires much memory
+to compile ; 4GB RAM or more is recommended. To build FreeOrion, expect up to
+45 minutes on a Intel Core i5 system.
+
+
+## Required Software Dependencies
+
+FreeOrion depends on the following software to build:
+
+  * [Visual Studio] - 2015 for Windows Desktop Update 3 ; Windows only
+  * [Xcode] - 6.4 or later ; Mac OS X only
+  * [CMake] - 3.4 (Mac OS X) ; 3.1 or later (Other non-Windows)
+  * A C++11 compliant compiler - Other Operating Systems
+    * [GNU GCC] - 5.0 or later
+    * [Clang] - 3.9 or later
+  * [Python] - 2.7 or later, but not 3 or later
+  * [Git]
+
+FreeOrion depends on the following libraries or APIs to run:
+
+  * OpenGL - 2.1 or later ; usually provided by the graphic card driver or
+    Operating System
+  * OpenGL Utilities (GLU) ; usually provided by the graphic card driver or
+    Operating System
+  * OpenAL - It's recommended to use the [OpenAL Soft] implementation
+  * [Boost] - 1.56 or later
+  * [zlib]
+  * [libpython] - 2.7 or later, but not 3 or later
+  * [FreeType2]
+  * [libpng]
+  * [libogg]
+  * [libvorbis]
+  * [SDL2]
+
+
+## Obtaining FreeOrion Source Code and Software Dependencies
+
+For Windows and Mac OS X a [FreeOrion Software Development Kit] is provided as
+download to compile FreeOrion from source. It contains the preconfigured and
+-compiled build and runtime dependencies for Visual Studio v140 toolchain on
+Windows and Mac OS X 10.9 SDK with Xcode or later on Mac OS X.
+
+For Linux or other Operating Systems the build and runtime dependencies should
+be installed by the preferred native way (e.g. via Package manager or compiling
+from source).
+
+Step by step procedure:
+
+ * On Windows or Mac OS X:
+   * Download the [FreeOrionSDK v8] from the FreeOrionSDK respository releases.
+ * Linux and other Operating Systems:
+   * Install build and runtime dependencies by the preferdded native way.
```

Comment:
See above.

Otherwise, if you want to stick with the "preferred native way" phrase, typo: preferdded -> preferred.

Reply:
Fixed.